app/views/devices_guidance/how_to_order_for_independent_special_schools.html.erb

```diff
@@ -0,0 +1,215 @@
+
+    <p class="govuk-body">
+      LAs can [request internet access for eligible pupils](#) too.
+    </p>
+
+    <h2 class="govuk-heading-l">
+      How to get laptops and internet access
+    </h2>
+
+    <h3 class="govuk-heading-m">
+      1. Check your local authority’s laptop allocation
+    </h3>
+
+    <p class="govuk-body">
+      DfE has asked each LA to nominate one or more staff members to order laptops and request internet access for eligible pupils.
+    </p>
+    <p class="govuk-body">
+      If you’ve been nominated to order, we’ll email your LA’s estimated laptop allocation to you. If you have not received
+      this email, <%= govuk_link_to 'contact us', support_ticket_path %>.
+    </p>
+    <p class="govuk-body">
+      Your estimate is based on the data your local authority submitted in the latest published alternative provision census (2020).
+    </p>
+    <p class="govuk-body">
+      You’re allocated one laptop for each pupil in years 3 to 11 that you reported as:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        having a state-funded place at an independent special school, or independent alternative provision
+      </li>
+      <li>
+        also being eligible for free school meals (FSM)
+      </li>
+    </ul>
+    <p class="govuk-body">
+      You can request internet access for these pupils if they need it too.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Tell us if your allocation is wrong
+    </h4>
+    <p class="govuk-body">
+      If your estimated allocation is wrong, <%= govuk_link_to 'contact us', support_ticket_path %> and let us know
+      how many laptops you currently need for eligible pupils. Do not include any personal data, such as pupils’ names or dates of birth.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      2. Check what pupils need
+    </h3>
+    <p class="govuk-body">
+      We recommend contacting pupils, families or schools as soon as possible to find out the following information.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Laptops
+    </h4>
+    <p class="govuk-body">
+      To order laptops, you need to find out which:
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        eligible pupils do not have access to a laptop or tablet
+      </li>
+      <li>
+        <%= govuk_link_to "Type of laptop and IT settings are most suitable for each care leaver", '/devices/device-specification' %>
+        - if they need Google Chromebooks, you should ask for their educational setting’s domain for
+        <span class="app-no-wrap">Google Workspace for Education Fundamentals</span>
+        <span class="app-no-wrap">(formerly G Suite for Education)</span>, and recovery email address. We’ll ask you for
+        these after you place your order.
+      </li>
+    </ul>
+    <p class="govuk-body">
+      <%= govuk_link_to 'Find out about safeguarding responsibilities, laptop ownership and providing IT support', '#step-6' %>.
```

Comment:
SC version of this says:
```
 See step 6 to find out about safeguarding responsibilities, laptop ownership and providing IT support.
 ```

Reply:
Please use the same content as the SC version. Thanks